plugins/auth/index.js

```diff
@@ -105,8 +117,25 @@ exports.register = (server, options, next) => {
             server.auth.strategy('token', 'jwt', {
                 key: pluginOptions.jwtPublicKey,
                 verifyOptions: {
-                    algorithms: [ALGORITHM],
-                    maxAge: EXPIRES_IN
+                    algorithms: [ALGORITHM]
+                },
+                validateFunc: (request, token, cb) => {
+                    if (!token.exp) {
+                        // Token is a user API token
+                        const userFactory = request.server.app.userFactory;
+
+                        return userFactory.get({ username: token.username })
+                        .then(user => user.validateToken(token.uuid))
+                        .then(() => {
+                            cb(null, true, token);
+                        })
+                        .catch(() => {
+                            cb(null, false, {});
+                        });
+                    }
+
+                    // Token is a 12h access token
```

Comment:
Notably, if the token is a 12h access token that has expired, it won't even make it to the validateFunc, so if it made it this far we don't need to check that it hasn't expired.